enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
-by the `PlacementBinding`'s overrides.
-
-### User Stories
-
-#### Story 1
-
-As a system administrator, I want to know the status of a rollout for a policy and what clusters have been updated.
-
-- **Summary**
-
-  - Add a `RolloutStatus` to the `Policy` status in the CRD.
+sync controllers on managed clusters watch the associated managed cluster namespace on the hub for policy updates.
+
+### API Design
+
+#### Rollout Strategy API Integration
+
+Open Cluster Management policies will now have a new field of `spec.rolloutStrategy` that is based on the
+[rollout strategy API](https://open-cluster-management.io/concepts/placement/#rollout-strategy). An example is shown
+below.
+
+```yaml
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: sample-policy
+  namespace: policy-namespace
+spec:
+  disabled: false
+  remediationAction: enforce
+  rolloutStrategy:               # <-- New field
+    type: ProgressivePerGroup
+    progressivePerGroup:
+      minSuccessTime: 5m
+        progressDeadline: 10m
+        maxFailures: 2%
+        ...
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        ...
+```
 
-- **Details**
+The default `spec.rolloutStrategy.type` will be `All` which is the same as the current behavior of deploying to all
+clusters.
 
-  - The `RolloutStatus` is added to the status on the replicated policy.
-  - The `RolloutStatus` is added on the root policy per-cluster and as an aggregated status.
+#### manualPerGroup Rollout Strategy
+
+In addition to the standard options in the
+[rollout strategy API](https://open-cluster-management.io/concepts/placement/#rollout-strategy), there will be an
+additional `spec.rolloutStrategy.type` option of `manualPerGroup` and will be contributed to the upstream rollout
+strategy API. This acts like `progressivePerGroup` except the user has manual control for when a group is available to
+receive the new policy version. The order of the groups in the `Placement` is not seen as a dependency chain but will
+act as a tiebreaker for the order of rollout when multiple groups are approved.
+
+This is to account for complex environment requirements that may not be conducive to automated rules. For example, a
+group of clusters that handle credit card machines are in a change freeze between the United States Thanksgiving holiday
+and Christmas due to peak shopping, however, other clusters may have low usage during that time and is an ideal period
+for deployments.
+
+To control these groups, the user will interact with a new Kubernetes resource of `Rollout` as shown below.
+
+```yaml
+apiVersion: v1beta1
+kind: Rollout
+metadata:
+  name: policy-<same as root policy>
+  namespace: <same as root policy>
+spec:
+  approvalsForVersion: "2.0" <-- optional if the user wants to tie the approval to a specific policy "version".
+  decisionGroups:
```

Comment:
So this is the order of groups?

Reply:
The order here doesn't actually matter. I prefer using a map in these cases but most Kubernetes APIs use lists with objects with `name` keys, so this copies that. The order in the placement matters in terms of priority but are not dependencies like in `progressivePerGroup`.